models/feedback.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+const shortid = require("shortid");
+
+const feedbackSchema = new mongoose.Schema({
+    _id: {
+        type: String,
+        default: shortid.generate,
+    },
+    timestamp: {
+        type: Date,
+        default: Date.now
+    },
+    userId: {
+        type: String,
+        required: true,
```

Comment:
@Kelvin-K here you can just add 
eg ref : "Users" to get the reference id from the model .

Reply:
Done